java/ql/integration-tests/all-platforms/java/buildless-sibling-projects/buildless-fetches.expected

```diff
@@ -1,3 +1,6 @@
+https://jcenter.bintray.com/junit/junit/4.12/junit-4.12.jar
+https://jcenter.bintray.com/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar
+https://jcenter.bintray.com/org/slf4j/slf4j-api/1.7.21/slf4j-api-1.7.21.jar
 https://repo.maven.apache.org/maven2/com/feiniaojin/naaf/naaf-graceful-response-example/1.0/naaf-graceful-response-example-1.0.jar
```

Comment:
Because the Gradle config uses JCenter -- note Git is being misleading by portraying this as a change; the `.actual` file was deleted, while the new expected file was created independently documenting the fetches made relating to the sibling projects